index.bs

```diff
@@ -1215,6 +1215,383 @@ implementation of this step is specific to the user agent.", the controlling
 user agent may send the remote-playback-termination-request message.
 
 
+Streaming Protocol {#streaming-protocol}
+========================================
+
+The section defines the use of the Open Screen Protocol for streaming
+media from one agent to another. We refer to the agent sending media
+as the *sender* and the agent receiving the media as the *receiver*.
+
```

Comment:
Maybe point out that a single agent can be a "sender" and "receiver" at the same time.

Reply:
Done